meta.jq

```diff
@@ -319,9 +319,52 @@ def build_command:
 
 			# TODO consider / check what "crane validate" does and if it would be appropriate here
 
-			# TODO generate SBOM? ... somehow
-
-			empty
+			if build_should_sbom then
```

Comment:
Ahhh, this is currently causing `mips64le` builds to fail because we didn't replicate the architecture check:

https://github.com/docker-library/meta-scripts/blob/f5b18c803010716859916e88cba38834f2b68a7f/meta.jq#L145-L148

Maybe that check should just be part of `doi.jq` in `build_should_sbom` directly instead?

Reply:
#73